tests/dragonfly/snapshot_test.py

```diff
@@ -217,6 +217,37 @@ async def test_snapshot(self, df_seeder_factory, async_client, df_server):
         assert super().get_main_file("test-set-snapshot_cron-summary.dfs")
 
 
+@dfly_args({**BASIC_ARGS})
+class TestDoubleSave(SnapshotTestBase):
```

Comment:
nit: rename test name to TestOnlyOneSaveAtATime
edit test comment: Dragonfly does not allow simultaneous save operations, send 2 save operations and make sure one is rejected